### PR TITLE
Add new cluster for pauld

### PR DESCRIPTION
**Pull request by pauldougan**
new improved cluster

### CODE REVIEW

File: terraform/clusters/pauld.run-sandbox.aws.ext.govsvc.uk/kubeconfig [hunk 1]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```
Comment (paroxp) [inline terraform/clusters/pauld.run-sandbox.aws.ext.govsvc.uk/kubeconfig]: Can we get rid of this file?

It's a pain to keep it up to date and should always be available through `terraform output`.

Comment (pauldougan) [reply terraform/clusters/pauld.run-sandbox.aws.ext.govsvc.uk/kubeconfig]: sure, I did wonder about this, will delete

Comment (pauldougan) [reply terraform/clusters/pauld.run-sandbox.aws.ext.govsvc.uk/kubeconfig]: done

File: terraform/clusters/pauld.run-sandbox.aws.ext.govsvc.uk/cluster.tf [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    cluster_name = "pauld"
+    dns_zone = "run-sandbox.aws.ext.govsvc.uk"
+    user_data_bucket_name = "gds-re-run-sandbox-terraform-state"
+    user_data_bucket_region = "eu-west-2"
+    k8s_tag = "v1.12.2"
+    admin_role_arns = ["arn:aws:iam::${data.aws_caller_identity.current.account_id}:role/admin"]
+
+    addons = {
+      ingress = 1
+      canary = 0
```
Comment (paroxp) [inline terraform/clusters/pauld.run-sandbox.aws.ext.govsvc.uk/cluster.tf]: Would you enable the canary? It's not strictly required, but is the only mean of monitoring for our clusters.

**Pull request merged**